python/py-catalogue/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        explosion catalogue 0.0.8 v
+revision            0
+name                py-${github.project}
+categories-append   devel
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Super lightweight function registries for your library
+
+long_description    catalogue is a tiny, zero-dependencies library\
+                    that makes it easy to add function (or object)\
+                    registries to your code. Function registries are\
+                    helpful when you have objects that need to be both\
+                    easily serializable and fully\
+                    customizable. Instead of passing a function into\
+                    your object, you pass in an identifier name, which\
+                    the object can use to lookup the function from the\
+                    registry. This makes the object easy to serialize,\
+                    because the name is a simple string. If you\
+                    instead saved the function, you'd have to use\
+                    Pickle for serialization, which has many drawbacks.
+
+checksums           rmd160  5516e535cff14b168f992c1fa06481d83ca74fb4 \
+                    sha256  8888b6f47a806e3a30d8c1c5049a8305cfcc0bc83301882d2daa66fed091e9eb \
+                    size    8310
+
+python.versions     35 36 37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-pytest
```

Comment:
@essandess I'm pretty sure that ```pytest``` is not a runtime dependency for this port... or? Additionally, you're missing a dependency on ```importlib_metadata``` for anything below PY38 ([see uptream's setup.cfg](https://github.com/explosion/catalogue/blob/6f44510c6c3e10216b99076f631f93cf2132be2f/setup.cfg#L34)), packaged in MacPorts under ```pyXY-importlib-metadata```. Please be more careful about dependencies as I've  noted in some of your other PRs as well.

Reply:
I'll edit this dependency in https://github.com/macports/macports-ports/pull/5862